install-dev/theme/views/configure.php

```diff
@@ -99,6 +99,21 @@
 		<?php echo $this->displayError('shop_timezone') ?>
 	</div>
 
+    <!-- Enable SSL -->
+    <div class="field clearfix">
+        <label class="aligned"><?php echo $this->translator->trans('Enable SSL', array(), 'Admin.Shopparameters.Feature'); ?></label>
```

Comment:
```suggestion
        <label class="aligned"><?php echo $this->translator->trans('Enable SSL', array(), 'Install'); ?></label>
```
Indeed it makes more sense! Thank you for the suggestion @LouiseBonnard :)